app/build.gradle

```diff
@@ -231,26 +231,39 @@ tasks.whenTaskAdded { task ->
 }
 
 // -------------------------------------------------------------------------------------------------
-// Adjust: Read token from locale file if it exists (Only release builds)
+// Secrets: add API secrets from config files into the BuildConfig instance.
+// We use files because they're easier to manage than environment variables.
 // -------------------------------------------------------------------------------------------------
-
 android.applicationVariants.all { variant ->
+    def SENTRY_FIELD = 'SENTRY_DSN'
+    def SENTRY_DEBUG_FILE_PATH = "${rootDir}/.sentry_dsn_debug"
+    def SENTRY_RELEASE_FILE_PATH = "${rootDir}/.sentry_dsn_release"
     def variantName = variant.getName()
 
-    print("Adjust token (" + variantName + "): ")
+    print("Sentry DSN (" + variantName + "): ")
+
+    def filePath
+    if (variantName.contains("Debug")) {
+        filePath = SENTRY_DEBUG_FILE_PATH
```

Comment:
Why is there a Debug sentry path?

Reply:
There are two sentry endpoints/keys: one for debug and one for release. Debug, I assume, is for us to test to make sure Sentry is working correctly. Release is release.

I separated the file paths so that it's harder to ship the wrong key.